client/jetpack-cloud/sections/partner-portal/primary/wpcom-atomic-hosting/card-content.tsx

```diff
@@ -109,7 +109,9 @@ export default function CardContent( {
 		return {
 			title: plan?.getTitle?.().toString() || '',
 			description: getProductTagline( planSlug ) || '',
-			price: formatCurrency( agencyProduct?.amount || 0, 'USD', { stripZeros: true } ),
+			price: formatCurrency( parseFloat( agencyProduct?.amount as string ) || 0, 'USD', {
+				stripZeros: true,
+			} ),
```

Comment:
[non-blocker] I noticed that we hardcode the currency here, maybe we can supply this value from the agencyProduct.

```suggestion
			price: formatCurrency(
				parseFloat( agencyProduct?.amount as string ) || 0,
				agencyProduct?.currency || 'USD',
				{
					stripZeros: true,
				}
			),
```

Reply:
Good catch! I made the suggestion: https://github.com/Automattic/wp-calypso/pull/84742/commits/343f487284c596c6d2d1584b33cf30324a9ea910